bigquery_etl/cli/backfill.py

```diff
@@ -522,14 +522,15 @@ def _initiate_backfill(
             destination_table=table,
             backfill_date=entry.start_date.isoformat(),
         )
-        custom_query = Path(query)
+        custom_query_path = Path(query)
         click.echo(
             click.style(
-                f"Starting backfill with custom query: '{custom_query}'.", fg="blue"
+                f"Starting backfill with custom query: '{custom_query_path}'.",
+                fg="blue",
             )
         )
-    elif entry.custom_query:
-        custom_query = Path(entry.custom_query)
+    elif entry.custom_query_path:
+        custom_query_path = Path(entry.custom_query_path)
```

Comment:
I would like to add 2 more options here:

fail or warn if `entry.shredder_mitigation is True and entry.custom_query_path` to warn people that only the `entry.shredder_mitigation` will take effect

and one if neither are present to fail